tmt/hardware.py

```diff
@@ -72,7 +73,6 @@
 # The default formatting should use unit symbols rather than full names.
 UNITS.default_format = '~'
 
-
```

Comment:
One immediate comment, there are way too many formatting changes, obscuring the actual changes. Empty lines added and removed, indentation changes... I wonder how that happens, we should be using the same `autopep8` settings in pre-commit.

The rest seems safely trivial, but need to re-read once the dust settles.

Reply:
Hmm, that must have been from my IDE, but I thought it was already setup to use `ruff-format`, so I'm not sure why this got changed and not in the `pre-commit`, I can review the commit more and minimize these